docs/units/distributions.rst

```diff
@@ -0,0 +1,243 @@
+.. |quantity| replace:: :class:`~astropy.units.Quantity`
+.. |distribution| replace:: :class:`~astropy.units.Distribution`
+
+.. _unit_distributions:
+
+Distributions
+*************
+
+In addition to |quantity|, astropy provides a |distribution| object to represent
+statistical distributions in a form that acts as a drop-in replacement for
+|quantity| or regulay Numpy arrays. USed in this manner, |distribution| provides
+uncertainty propogation at the cost of additional computation.  It can also
+more generally represent sampled distributions for e.g., Monte Carlo calculation
+techniques.
+
+The core object for this feature is the |distribution|.  Currently, all
+such distributions are Monte Carlo sampled.  While this means each distribution

```

Comment:
Here and maybe elsewhere, hyphenate Monte-Carlo

Reply:
I disagree.  In almost all stats literature I'm familiar with you don't hyphenate it.  As a (admittedly slightly flimsy but still useful) source: https://en.wikipedia.org/wiki/Monte_Carlo_method

That said... this *might* be a British vs American English thing?  Maybe we need a cross-Atlantic astropy style guide :wink: